### ISSUE TITLE
How to run predictions on saved images/drawings ?

### ISSUE THREAD

**Issue by shubhank008** [zaidalyafeai/zaidalyafeai.github.io#19]
Tried to make a prediction on a .jpg/.png saved image of random drawings (from quick draw data set, directly saved the image).
However, the prediction came off quite wrong and can't seem to get it working and my guess is on due to the image processing before feeding to the model.  

This is what I am using

```
from PIL import Image
import cv2
import matplotlib.pyplot as plt
from random import randint
%matplotlib inline  

clock = qd.get_drawing("clock")
apple = clock

img = apple.image.convert("L")
img = img.resize((28,28),Image.BILINEAR)
img = np.array(img)
img = cv2.bitwise_not(img)


img = img.reshape(28,28,1)
img = img.astype('float32')
img /= 255.0

print(img.shape)
plt.imshow(img.squeeze()) 
print(img)
```

Yellowish image is the original one in npy format, the other is processed using above code.

<img width="345" alt="chrome_2019-06-06_10-01-14" src="https://user-images.githubusercontent.com/3790163/59006952-17cc6100-8842-11e9-9c24-c4e5850143ac.png">

<img width="406" alt="chrome_2019-06-06_10-01-27" src="https://user-images.githubusercontent.com/3790163/59006953-1864f780-8842-11e9-9318-24f20492a7c0.png">

**Comment by zaidalyafeai** [comment]
@shubhank008, can you explain more ? which model are you using ? Are you using this [notebook](https://colab.research.google.com/github/zaidalyafeai/zaidalyafeai.github.io/blob/master/sketcher/Sketcher.ipynb) ?

**Comment by shubhank008** [comment]
> @shubhank008, can you explain more ? which model are you using ? Are you using this [notebook](https://colab.research.google.com/github/zaidalyafeai/zaidalyafeai.github.io/blob/master/sketcher/Sketcher.ipynb) ?

Yes, exactly that. Initially I tried to re-write how you were doing inference on drawings in web app (tensorflowjs) to python (using saved drawing images .jpg .png) but that did not work, then I started trying various image pre-processing alternatives as I am pretty sure its the whole image reading, processing and feeding part that I am messing up.

**Comment by shubhank008** [comment]
@zaidalyafeai  anything ?

**Comment by zyqq** [comment]
Did you find a solution?
